projects/storefrontlib/shared/components/card/card.component.ts

```diff
@@ -21,6 +21,7 @@ export interface Card {
   actions?: Array<CardAction | CardLinkAction>;
   deleteMsg?: string;
   label?: string;
+  isRegion?: boolean;
```

Comment:
Why do we need to limit `role` to `region` only? Couldn't we have this as a `string` or even better `enum` with role types?

Reply:
I will explore this